### ISSUE TITLE
NTR: Arthrocentesis: positive routine culture (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8077]
New term request Arthrocentesis: positive routine culture (hausma_130801023246): Joint (synovial) tap;  UMLS: 0368552

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000582
- label: Positive synonvial fluid culture
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: 
- def: Growth of an infectious agent in a culture derived from synovial fluid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8077
- synonyms:

**Comment by MickeySegal** [comment]
I added the word bacterial to our term for clarity.
